features/org.wso2.analytics.is.feature/src/main/siddhi4/README.md

```diff
@@ -1,5 +1,32 @@
-
 Download SP4.2.0 pack.
 Start the worker profile using, ./bin/worker.sh from <SP_HOME>
 Drop the siddhi files in <SP_HOME>/wso2/worker/deployment/siddhi-files/
-To deploy The To deploy the siddhi4 execution files refer link https://docs.wso2.com/display/SP420/Deploying+Streaming+Applications
\ No newline at end of file
+To deploy The To deploy the siddhi4 execution files refer link https://docs.wso2.com/display/SP420/Deploying+Streaming+Applications
+
+**For authentication Data Analytics:**
+- Download WSO2 stream Processor (This will be referred as <SP_HOME>)
+- Open the file "<SP_Home>/conf/worker/deployment.yaml" and add following datasource:
+    ```
+    - name: IS_DATA_DUMP_DB
```

Comment:
How about we change it to IS_ANALYTICS_DB?

Reply:
Okay. Will change the name.